packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -4,85 +4,65 @@
 import {
 	store as coreStore,
 	useResourcePermissions,
+	useEntityRecords,
 } from '@wordpress/core-data';
 import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
-import { SELECT_NAVIGATION_MENUS_ARGS } from './constants';
+import { PRELOADED_NAVIGATION_MENUS_QUERY } from './constants';
 
 export default function useNavigationMenu( ref ) {
 	const permissions = useResourcePermissions( 'navigation', ref );
 
-	return useSelect(
+	const useSelectResult = useSelect(
```

Comment:
I think you should just extract the vars here. This `const` isn't a typical pattern and it's not even warranted now the result of the `useSelect` is much reduced by recent changes to this PR.

Reply:
good idea, done